lib/daisy_ui_components/tabs.ex

```diff
@@ -0,0 +1,250 @@
+defmodule DaisyUIComponents.Tabs do
+  @moduledoc """
+  Tab component
+
+  Renders a list of tab links or toggles with radio toggle and content or label with radio toggle and content
+
+  https://daisyui.com/components/tab/
+  """
+
+  use DaisyUIComponents, :component
+
+  @doc """
+  Renders a Tabs with links as toggle by default.
+
+  ## Default Example
+  <.tabs>
+    <:tab>Tab 1</:tab>
+    <:tab active>Tab 2</:tab>
+    <:tab disabled>Tab 3</:tab>
+  </.tabs>
+
+  ## Renders
+  <div role="tablist" class="tabs">
+    <a role="tab" class="tab">Tab 1</a>
+    <a role="tab" class="tab tab-active">Tab 2</a>
+    <a role="tab" class="tab tab-disabled">Tab 3</a>
+  </div>
+
+  ## Radio Example
+  ## https://daisyui.com/components/tab/#radio-tabs-border--tab-content
+  <.tabs type="radio">
+    <:tab>Tab 1</:tab>
+    <:tab active>Tab 2</:tab>
+    <:tab>Tab 3</:tab>
+  </.tabs>
+
+  ## Renders
+  <div class="tabs">
+    <input type="radio" name="tabs" class="tab" aria-label="Tab 1">
+    <input type="radio" name="tabs" class="tab" aria-label="Tab 2" checked="checked">
+    <input type="radio" name="tabs" class="tab" aria-label="Tab 3">
+  </div>
+
+  ## Label Example
+  ## https://daisyui.com/components/tab/#radio-tabs-lift-with-icons--tab-content
+  <.tabs type="label">
+    <:tab>Tab 1</:tab>
+    <:tab active>Tab 2</:tab>
+    <:tab>Tab 3</:tab>
+  </.tabs>
+
+  ## Renders
+  <div class="tabs">
+    <label class="tab">
+      <input type="radio" name="tabs" class="tab"> Tab 1
```

Comment:
From what I saw in the docs, should use aria-label as text and not the slot
```suggestion
      <input type="radio" name="tabs" class="tab"  aria-label="Tab 1" />
```

Reply:
From the docs example, the radio toggle only example puts the title into the `aria-label`. From [this example](https://daisyui.com/components/tab/#radio-tabs-lift-with-icons--tab-content) (with labels) it looks like the label wraps the radio toggle with the title rendered separate.